### ISSUE TITLE
Conflict between eslint and prettier in alpha 28

### ISSUE THREAD

**Issue by pbauer** [plone/volto#5238]
**Describe the bug**
The new version of prettier in Volto 17.0.0-alpha.28 generates different styling from eslint.

Add the following file `Example.jsx` to a fresh project created with `yo @plone/volto --canary`:

```jsx
import { Component } from 'react';
import qs from 'query-string';

class Example extends Component {
  constructor(props) {
    const searchableText = qs.parse(props.history.location.search).SearchableText;
    if (searchableText) {
      // do stuff
    }
  }
}

export default Example;
```

`yarn lint:fix` creates:
```jsx
import { Component } from 'react';
import qs from 'query-string';

class Example extends Component {
  constructor(props) {
    const searchableText = qs.parse(
      props.history.location.search,
    ).SearchableText;
    if (searchableText) {
      // do stuff
    }
  }
}

export default Example;
```

`yarn prettier:fix` creates
```jsx
import { Component } from 'react';
import qs from 'query-string';

class Example extends Component {
  constructor(props) {
    const searchableText = qs.parse(props.history.location.search)
      .SearchableText;
    if (searchableText) {
      // do stuff
    }
  }
}

export default Example;
```

Which lint will change again and then prettier and again and again.
So lint and prettier will run in circles and ci is never green.

**Comment by pbauer** [comment]
Fixed by updating prettier from 2.0.5 to 3.0.3. It turns our my generator picket the newest volto version but still added outdated version-pinns among them `"prettier": "2.0.5",`

**Comment by pbauer** [comment]
After updating the generator (`npm install -g @plone/generator-volto@7.0.0-alpha.10`) the versions were fine. Sorry for the noise.
